test/e2e/kubectl.go

```diff
@@ -583,7 +583,6 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			_, err := framework.RunHostCmd(ns, simplePodName, "/kubectl get pods --token=invalid --v=7 2>&1")
 			framework.Logf("got err %v", err)
 			Expect(err).To(HaveOccurred())
-			Expect(err).To(ContainSubstring("User \"system:anonymous\" cannot list pods in the namespace"))
```

Comment:
Removing this check is fine with me, though an invalid token should return a 401 Unauthorized error, not a 403 Forbidden. That can be a follow up issue and assigned to me